.github/workflows/deploy-unsigned.yml

```diff
@@ -138,10 +161,10 @@ jobs:
 
       - name: "Notify Slack: React to top-level notification"
         if: always()
-        uses: archive/github-actions-slack@v2.0.0
+        uses: archive/github-actions-slack@v2.6.0
         with:
           slack-function: send-reaction
           slack-bot-user-oauth-access-token: ${{ secrets.SLACK_BOT_USER_OAUTH_ACCESS_TOKEN }}
           slack-channel: ${{ env.SLACK_CHANNEL }}
           slack-message-timestamp: ${{ needs.publish-images.outputs.slack-response-ts }}
-          slack-emoji-name: ${{ env.DEPLOY_STATE_EMOJI }}
+          slack-emoji-name: ${{ env.DEPLOY_STATE_EMOJI }}
```

Comment:
Did you test the happy path, where `env.DEPLOY_STATE_TEXT` and `env.DEPLOY_STATE_EMOJI` are set to a "success" state in the `deploy-images` job? I'm thinking since those env variables get set in that job, the `notify-slack` job won't see those updates. If so, I think you'll have to output those success values or simply check if both jobs passed.

Reply:
understood.
i updated the code and it seems to be working now :)